cli/src/deno.rs

```diff
@@ -49,12 +49,8 @@ pub async fn run(
     let main_module =
         deno_core::resolve_path(&extension.entry_point().to_string_lossy(), &PathBuf::from("."))?;
 
-    let bootstrap = BootstrapOptions {
-        args,
-        runtime_version: env!("CARGO_PKG_VERSION").into(),
```

Comment:
This was putting the Phylum CLI version in `Deno.version.deno`. That variable is now blank (set [here][1] in the `deno_runtime` snapshot).

[1]: https://github.com/denoland/deno/blob/9444bab18647bc871b63716a3a70b2aebaaf3821/runtime/build.rs#L38-L40

We could consider injecting the Phylum CLI version into the extension runtime somewhere else. But that would be a separate PR.